sentences/de/_common.yaml

```diff
@@ -161,9 +161,10 @@ expansion_rules:
   szene: "[die] Szene"
   skript: "[das] Skript"
   batterie: "[die|der|des] (Batterie[n]|Akku[s])"
-  irgend: "(irgend(ein[e][s|r]|welche[s])|einige[s]) [der]"
+  ein: "ein[e|er|es|s]"
+  irgend: "(irgend(ein[e][s|r]|welche[s])|(einige|manche)[s]|<ein>) [der]"
```

Comment:
```suggestion
  irgend: "(irgend(<ein>|welche[s])|(einige|manche)[s]|<ein>) [der]"
```

Reply:
Hi @mib1185 I had it that way first but the tests were failing. It seems like it currently doesn't work when you use the same expansion rule twice in one line ... 😞

Reply:
I'll set it back to ready for review. I don't have an idea how to restructure it to make it working with just the expansion rule ... I also had the same issue somewhere else.

Reply:
created a fix for this tests #1398 - when this fix is accepted we can rebase the current branch and use `<ein>` multiple times

Reply:
@mib1185 how should we proceed? Merge this PR and once there is a solution use `<ein>` multiple times? Functionality doesn't change so maybe that's the best option?

Reply:
I requested review again. I'd change the rule when there is a fix available.